packages/Table/tests/spec/Table.spec.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Table from "../../src/Table";
+
+const data = [
+  { name: "Charles ", lastName: "Babbage" },
+  { name: "Claude ", lastName: "Shannon" },
+  { name: "Alan ", lastName: "Turing" },
+  { name: "John  von", lastName: " Neumann" },
+  { name: "William", lastName: "Shockley" },
+  { name: "Douglas", lastName: "Engelbart" },
+  { name: "Robert", lastName: "Noyce" },
+  { name: "Steve", lastName: "Wozniak" },
+  { name: "Grace Murray", lastName: "Hopper" },
+  { name: "Vint", lastName: "Cerf" },
+];
+
+function renderComponent() {
+  return render(
+    <Table data={data}>
+      <Table.ColumnDefinition header="Name" cell="name" />
+      <Table.ColumnDefinition header="LastName" cell="lastName" />
+    </Table>
+  );
+}
+
+describe("Table", () => {
+  it("Should render data in the table component", () => {
+    const { getByText } = renderComponent();
+
+    expect(getByText("Name")).toBeVisible();
```

Comment:
we can do the same than with cypress
```js
["name1", "name2", "..."].forEach(name => {
      expect(getByText(name)).toBeVisible();
})
```
we are duplicating testing here and with cypress, there is no harm in adding more test, but we should be aware if we are duplicating effort, it's ok for now :)